### PR TITLE
Examples added for adding and removing elements in a FiniteSet

### PR DESCRIPTION
**Pull request by Yathartha22**
I thought to add few examples in `FiniteSet` class of how one can add or remove elements from a particular `FiniteSet`. Therefore making it easy to understand it. These examples also shows the functionality of `FiniteSet`. 

I faced difficulty on this particular thing.  So I thought rather add it in the docs so that in future anyone won't waste time looking for it.

### CODE REVIEW

**Comment by Yathartha22** [comment]
I don't know why the test build failed. It is strange that the first commit passed(which had wrong braces), while the other failed(which had the right ones.)

@asmeurer @smichr

**Comment by Yathartha22** [comment]
@asmeurer @jksuom @smichr can you please review it.

**Review by siefkenj — APPROVED** [review]

**Comment by Yathartha22** [comment]
Can it be merged ?

File: sympy/sets/sets.py
Comment (smichr) [inline sympy/sets/sets.py]: How about 

```
>>> f = FiniteSet(*members); f
{1, 2, 3, 4}
>>> f - FiniteSet(2)
{1, 3, 4}
>>> f + FiniteSet(5, 0)
{0, 1, 2, 3, 4, 5}
```

(Not a big deal, just a little more compact.)

Comment (Yathartha22) [inline sympy/sets/sets.py]: @smichr yeah this looks more simplified and appropriate.
I will make the necessary changes.
Thanks!!

**Comment by kshitij10496** [comment]
@Yathartha22 Can you squash the commits into a single commit ?
This should be good to go once the tests pass.

**Comment by Yathartha22** [comment]
@kshitij10496 I will do it.

**Comment by Yathartha22** [comment]
I squashed it, but don't know why still two commits are being shown.
I tried to squash it again, but it  shows only one commit in the git bash.

**Comment by kshitij10496** [comment]
I hope you are using interactive rebasing to squash the commits.
Read [this](http://stackoverflow.com/q/5189560/4927751) as reference.

`git rebase -i HEAD~2` ought to be the starting point to squash the previous two commits.

**Comment by Yathartha22** [comment]
thanks @kshitij10496

**Comment by Yathartha22** [comment]
@smichr @kshitij10496 is this okay?

**Comment by kshitij10496** [comment]
LGTM +1
@smichr If this PR has your approval, then lets merge it.

**Comment by Yathartha22** [comment]
thanks @smichr  @kshitij10496

**Pull request merged**
